fluent-react/src/context.ts

```diff
@@ -1,4 +1,10 @@
 import { createContext } from "react";
 import { ReactLocalization } from "./localization";
 
-export let FluentContext = createContext(new ReactLocalization([], null));
+const defaultValue = {
+  l10n: new ReactLocalization([], null),
+  changeLocales: (_changeLocales: string[]) => undefined as void,
+  currentLocales: [] as string[],
```

Comment:
Since we have a setter and a reader, maybe we could call they as `locales` and `setLocales`?